acceptance/tests/puppetserver_requests.rb

```diff
@@ -0,0 +1,26 @@
+require 'beaker-http'
+
+test_name 'Ensure that requests can be built out to the puppetserver'
+
+step 'install latest released puppet agent' do
+  install_puppet_agent_on(hosts)
+end
+
+step 'install the latest puppet-server on the master' do
+  install_package(master, 'puppetserver')
+  on master, 'service puppetserver start'
```

Comment:
hmmm... isn't there some faster way to test an http connection object than to install and start puppetserver?  just go get the weather or something